tests/backup/backup_helper.go

```diff
@@ -250,6 +250,13 @@ var (
 				},
 			},
 		},
+		"postgres-csi": {
+			PreRule: backup.PreRule{
+				Rule: backup.RuleSpec{
+					ActionList: []string{"PGPASSWORD=$POSTGRES_PASSWORD; psql -U \"$POSTGRES_USER\" -c \"CHECKPOINT\""}, PodSelectorList: []string{"app=postgres"}, Background: []string{"false"}, RunInSinglePod: []string{"false"}, Container: []string{},
```

Comment:
It should have been []bool{false} instead of []string{"false"}; may be in a different PR @mkoppal-px

Reply:
@kphalgun-px The fix will be required for all the existing rules https://github.com/portworx/torpedo/blob/403eb4309a88b531909255fcedb1d55d3b7a1853/tests/backup/backup_helper.go#L235

Reply:
Yes. Hopefully someone can fix it in the future.